docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,249 @@
+Restoring the previous version will preserve the current version, then load the changes from the version you're restoring. Before restoration, you may want to view the changes in the previous version. To do this, select the **Peek** option, which opens the previous version in a new tab.
+
+![restore-version.png](imgs/restore-version.png)
+
+### Snapshots
+
+Snapshots are automatically saved as you build.
+
+:::info
+
+Users on the *Free* and *Standard* plans can access only one day of snapshots.
+
+:::
+
+## 7. Project comments
+
+This helps you leave your thoughts, opinions, questions, or feedback on a specific widget with your project team or a client. While adding a comment, you can tag users, and they will be able to respond, creating a thread of conversation.
+
+:::info
+
+To tag users, select the @ symbol and choose the project team member(s).
+
+:::
+
+## 8. Optimizations & Enhancements
+
+We scan and suggest enhancements to elevate your app's design and speed. Imagine having a personal consultant for design and performance dedicated to improving the user experience of your app.
+
+**Optimizations**: We spotlight elements that may slow down your app. For example, a query on a column, unused and duplicate backend query, a widget with unbounded size, etc.
+
+**UI Enhancements**: We offer tips for a more visually appealing and user-friendly design, like resizing a small tap target of any widget.
+
+:::info
+
+You can control what kind of suggestions you would like to receive by clicking on the settings icon on the right.
+
+:::
+
+![Optimization.avif](imgs/Optimization.avif)
+
+## 9. Project issues
+
+If there are any issues or warnings present in your current project that might result in a build failure or app crash, they will be displayed in this section. You can click this option to view a short description of each issue and navigate to the place where you can fix the issue by clicking on the respective issue.
+
+Errors prevent your app from compiling and running. These must be resolved in order to run the app. They can be due to missing actions, errors in custom code, incorrect data types, and so on.
+
+Warnings, while not preventing compilation, indicate potential issues such as incorrect rules setup or performance problems. Although it's possible to ignore warnings, addressing them can enhance the quality of your app and prevent future issues.
+
+![Warning-and-errors.avif](imgs/Warning-and-errors.avif)
+
+## 10. Branching
+
+[Branching](#) allows you to create a separate copy of your existing project to build new things without breaking existing features.
+
+## 11. Developer menu
+
+From here, you can view the code for your project, connect to GitHub, and also download the source code for your project.
+
+### 11.1 View Code
+
+This option lets you display the *Dart* code for all the pages of your FlutterFlow project. You can also take a look at the dependencies being used by the app here.
+
+### 11.2 Connect GitHub Repo (Paid Feature)
```

Comment:
Same comment about "Paid feature" being part of Admonition instead. Also add hyperlinks to "paid plan" and link to the pricing page. Hyperlink text should be bold as per the Docs style guide

Reply:
Added this into a single admonition.